androidApp/src/main/java/press/home/HomeView.kt

```diff
@@ -113,10 +124,58 @@ class HomeView @AssistedInject constructor(
     )
   }
 
+  private val createEditorView = { noteUuid: Uuid ->
+    editorViewFactory.create(
+        context = context,
+        openMode = ExistingNote(noteUuid),
+        onDismiss = { notesList.collapse() }
+    )
+  }
+
+  private var noteForEditor by observable<NoteParcelable?>(null) { _, oldNote, newNote ->
+    if (oldNote == newNote) return@observable
```

Comment:
We shouldn't be doing this as it'll be masking the actual problem that lies somewhere else. If the same note is being applied multiple times, the actual problem should be fixed.

Reply:
@saket I see. The original purpose is to prevent race event. But thinking about this now you are right about it hiding issue else where.